src/Components/UpdatePasswordForm.php

```diff
@@ -47,7 +47,7 @@ public function updatePassword(UpdatesUserPasswords $updater)
             'password_confirmation' => '',
         ];
 
-        $this->emit('saved');
+        $this->dispatchBrowserEvent('saved');
```

Comment:
make it a specific event, `saved` could be emitted by any component and would interfere